src/auditor/extra_checks.jl

```diff
@@ -0,0 +1,16 @@
+# This file contains some extra checks that don't fall into the other
+# categories, for example because they're very platform-specific.
+
+check_extra(::ObjectHandle, ::Platform, rest...; verbose::Bool = false, kwargs...) = true
+
+function check_extra(oh::ELFHandle, ::FreeBSD, rest...; verbose::Bool = false, kwargs...)
```

Comment:
Instead of calling this `check_extra`, I think we should call this `check_os_abi()`.  We can still put it in `extra_checks.jl`, but the function name should be different, IMO.